back/api/v1/validators.go

```diff
@@ -0,0 +1,76 @@
+package v1
+
+import (
+	"errors"
+	"net/mail"
+	"reflect"
+
+	"gopkg.in/validator.v2"
+)
+
+func validatePassword(v interface{}, param string) error {
+	st := reflect.ValueOf(v)
+	if st.Kind() != reflect.String {
+		return errors.New("password must be string")
+	}
+
+	// Password must be at least 12 characters long
+	if len(st.String()) < 12 {
+		return errors.New("password must be at least 12 characters long")
+	}
+
+	// Password must contain :
```

Comment:
This is not recommanded to force people to respect those criterias. Only recommand them to do so

Reply:
Then this should be a responsibility of the frontend to implement, I'll remove that part